README.md

```diff
@@ -459,6 +459,107 @@ for (const transactionSummary of transactionSummaries) {
+    sender: new AccountAddress(senderAccountAddress),
+};
+
+const deployModuleTransaction: AccountTransaction = {
+    header: header,
+    payload: deployModule as AccountTransactionPayload,
+    type: AccountTransactionType.DeployModule,
+};
+```
+
+Finally, to actually deploy the module to the chain, send the constructed `deployModuleTransaction` to the chain using `sendAccountTransaction`. (See [Send Account Transaction](#Send-Account-Transaction) for how to do this)
+
+## Init Contract (parameterless smart contract)
+The following example demonstrates how to initialize a smart contract from a module, which has already been deployed. 
+The name of the contract "INDBank".
+In this example, the contract does not take any parameters, so we can leave params as an empty Buffer.  
+```js
+const contractName = 'INDBank'; 
+const params = Buffer.from([]);
+//The amount of energy that can be used for contract execution.
+const maxContractExecutionEnergy = 300000n;
+```
+
+Create init contract transaction
+```js
+const initModule: InitContractPayload = {
+    amount: new GtuAmount(0n), // Amount to send to the contract. If the smart contract is not payable, set the amount to 0.
+    moduleRef: new ModuleReference('a225a5aeb0a5cf9bbc59209e15df030e8cc2c17b8dba08c4bf59f80edaedd8b1'), // Module reference, which can be obtained after deploying a module
+    contractName: contractName,
+    parameter: params,
+    maxContractExecutionEnergy: maxContractExecutionEnergy
+};
+
+const initContractTransaction: AccountTransaction = {
+    header: header,
+    payload: initModule,
+    type: AccountTransactionType.InitializeSmartContractInstance,
+};
+```
+
+Finally, to actually initialize the contract on the chain, send the constructed `initContractTransaction` to the chain using `sendAccountTransaction`. (See [Send Account Transaction](#Send-Account-Transaction) for how to do this)
+
+## Update Contract(parameterless smart contract)
+The following example demonstrates how to update a smart contract. 
+
+To update a smart contract we create a 'updateContractTransaction'.
+To do this we need to specify the name of the receive function, which should contain the contract name as a prefix ( So if the contract has the name "INDBank" and the receive function has the name "insertAmount" then the receiveName should be "INDBank.insertAmount").
+
+We also need to supply the contract address of the contract instance. This consists of an index and a subindex.
+
+In this example, the contract does not take any parameters, so we can leave the parameters as an empty list.  
+```js
+const receiveName = 'INDBank.insertAmount';
+const params = Buffer.from([]);
+const contractAddress = { index: BigInt(83), subindex: BigInt(0) } as ContractAddress;
+//The amount of energy that can be used for contract execution.
+const maxContractExecutionEnergy = 30000n;
+```
+Create update contract transaction
+```js
+const updateContractTransaction: UpdateContractPayload =
```

Comment:
```suggestion
const updateModule: UpdateContractPayload =
```
Currently it conflicts with the `AccountTransaction`'s name, and `updateModule` is the name used when constructing the `AccountTransaction`.